jgrapht-core/src/main/java/org/jgrapht/traverse/LexBreadthFirstIterator.java

```diff
@@ -74,6 +74,18 @@ public LexBreadthFirstIterator(Graph<V, E> graph) {
         GraphTests.requireUndirected(graph);
         bucketList = new BucketList(graph.vertexSet());
     }
+    
+    /**
+     * Creates new lexicographical breadth-first iterator with a static priority list for {@code graph}.
+     *
+     * @param graph the graph to be iterated.
+     * @param priority the vertex array sorted by their priorities.
+     */
+    public LexBreadthFirstIterator(Graph<V, E> graph, V[] priority) {
```

Comment:
Do we maybe want to check, if the priority is a valid priority of graph? i.e. check if graph.vertices and priority have the same vertices.